vehicle/as/CMakeLists.txt

```diff
@@ -1,23 +1,26 @@
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 3.5)
 project(as)
 add_compile_options(-std=c++14)
 
-find_package(
-  catkin REQUIRED COMPONENTS
-  roscpp
-  message_filters
-  std_msgs
-  geometry_msgs
-  automotive_platform_msgs
-  automotive_navigation_msgs
-  pacmod_msgs
-  autoware_control_msgs
-  autoware_vehicle_msgs
+find_package(ament_cmake REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(message_filters REQUIRED)
+find_package(std_msgs REQUIRED)
+find_package(geometry_msgs REQUIRED)
+find_package(automotive_platform_msgs REQUIRED)
+find_package(automotive_navigation_msgs REQUIRED)
+find_package(pacmod_msgs REQUIRED)
+find_package(autoware_control_msgs REQUIRED)
+find_package(autoware_vehicle_msgs REQUIRED)
+
+
+include_directories(
+  include
+  ${catkin_INCLUDE_DIRS}
```

Comment:
`${catkin_INCLUDE_DIRS}` doesn't exist anymore. I'm still learning about best CMake practices, but wouldn't you need to specify the individual include dirs, like [here](https://gist.github.com/dirk-thomas/83ae6bfbfc94e06cd519)?

Reply:
replaced by ament_auto.